lib/datadog/tracing/transport/trace_formatter.rb

```diff
@@ -203,6 +226,14 @@ def find_root_span(trace)
           # when root span is not found, fall back to last span (partial flush)
           root_span || trace.spans.last
         end
+
+        def git_repository_url
+          Core::Environment::Git.git_repository_url
+        end
+
+        def git_commit_sha
+          Core::Environment::Git.git_commit_sha
+        end
```

Comment:
Minor: Maybe it's just me, but it seems weird to add ~8 lines of code to be able to write `git_repository_url/git_commit_sha` instead of directly calling `Core::Environment::Git.git_repository_url/git_commit_sha` above? :eyes:

Reply:
yes, it is not worth it to add these methods

About the points above: we reviewed this with @TonyCTHsu and he ack'd this feature from Tracing side
I will backport it to master as soon as I merge it to 2.0 later today

Reply:
no, actually I found the reason I did
this looks awful:

```ruby
        def tag_git_repository_url!
          return if Core::Environment::Git.git_repository_url.nil?

          first_span.set_tag(Core::Git::Ext::TAG_REPOSITORY_URL, Core::Environment::Git.git_repository_url)
        end
```

Reply:
Still looks better than having two extra methods IMHO but I'll leave the final decision up to you since it's a very small detail ;)